src/runtime/vendor/github.com/kata-containers/govmm/qemu/qemu.go

```diff
@@ -2439,6 +2439,13 @@ type Config struct {
 	// Ctx is the context used when launching qemu.
```

Comment:
please raise a PR in https://github.com/kata-containers/govmm/ for this change

Reply:
@devimc, thanks! PR raised: https://github.com/kata-containers/govmm/pull/197

Reply:
For the sake of this PR (review), can you make the vendor hack/addition a separate commit?

Reply:
@egernst I just did as suggested. Can you please approve the workflow to run?